### PR TITLE
Day 3: To Do list: add & remove

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

### Project requirements

- [x] Remove all hardcoded items from the tasks array.
- [x] Create a new JavaScript file for the new functionality.
- [x] Implement a function for adding a new task (add a new element to the array).
- [x] Implement a function for editing task descriptions.
- [x] Implement a function for deleting a task (remove an element from the array).
- [x] Implement a function for the "Clear all completed" button (use filter() method).
- [x] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- [x] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
- [x] All changes to the To-Do List should be saved in local storage.

### Pair Programming 🎳

For this exercise, PP was not required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The following tools help make easier to work with sample code.
 
 ## Authors
```
Comment (carlos-ssh) [inline README.md:60]: Kindly work on these changes:
- Implement a function for the "Clear all completed" button (`use filter()` method).
- When i try to remove a task from the delete icon doesn't works well. i hit multiple time at this button and i had this error displayed:

Screen Shot 2021-12-04 at 9 27 36

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @benjp009 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (benjp009) [reply README.md:60]: Done - thanks for the assistance, I have changed some things on the list so it can work

**Comment by benjp009** [comment]
added .gitignore for node modules

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ The following tools help make easier to work with sample code.
...
 
-👤 **Benjamin**
+👤 **Benjamin** 
 
 - GitHub: [@benjp009](https://github.com/benjp009)
```
Comment (Mihndim2020) [inline README.md:65]: - [x] Please kindly delete the `node_modules` directory from the repository and add it to the `.gitignore` file as it is not necessary and it make your repository to be crowed.

dir

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @benjp009, 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there

 # Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjp009** [comment]
@Mihndim2020 I have made the changes, previously made on main directly
lol

**Review by ArthurGC — APPROVED** [review]
# Approved project!  💯 ✅ 🥳 🎈 🎉
Hi  @benjp009 

Amazing job implementing suggestions from the previous reviewer! 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**